### PR TITLE
Panel background color

### LINKED ISSUE #25089: [Bug]: Panel content overlays header and footer if panel is opened from start
### Library

React / v8 (@fluentui/react)

### System Info

```shell
System:
    OS: Windows 10 10.0.19044
    CPU: (16) x64 11th Gen Intel(R) Core(TM) i7-11800H @ 2.30GHz
    Memory: 44.17 GB / 63.73 GB
  Browsers:
    Edge: Spartan (44.19041.1266.0), Chromium (105.0.1343.53)
    Internet Explorer: 11.0.19041.1566
```

### Are you reporting Accessibility issue?

no

### Reproduction

https://codepen.io/chaoticgoodyury/pen/mdLjwMV

## Actual Behavior
Big and scrollable panel content overflows header and footer if Panel is opened from start.
image
The problem is resolved when window is resized or panel reopened, so I assume that there is some minimal state mismatch for the Panel component.

## Expected Behavior
Panel's header and footer are not overlayed by content when scrolling.

### Logs

_No response_

### Requested priority

Normal

### Products/sites affected

_No response_

### Are you willing to submit a PR to fix?

no

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

### PR DESCRIPTION
**Pull request by natalieethell**
## PR Description

A PR in April made changes to the Panel styles: https://github.com/microsoft/fluentui/pull/22408/files. 

Before that change, the Panel header always inherited the background color from its parent. Now, it only gets a background color when `isStickyFooter` is true.

However, `isStickyFooter` defaults to false until a resize occurs, so we are seeing bugs in our product where content shows behind the sticky header on scroll.

PanelOverlapBug

This change ensures that the header always receives a background color so it is not transparent.

## Related Issue(s)

Fixes #25089

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/b75de6cbb07fae634938821eb1afef450e1e46ef'>b75de6cbb07fae634938821eb1afef450e1e46ef</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)

  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>GroupedList</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/GroupedList-mount_base.html">2269</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/GroupedList-mount.html">2586</a></td>
            <td>2</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/GroupedList-mount.regression.txt">Possible regression</a></td>
           </tr></table><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>BaseButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/BaseButton-mount_base.html">1410</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/BaseButton-mount.html">1460</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Breadcrumb</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/Breadcrumb-mount_base.html">3477</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/Breadcrumb-mount.html">3435</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Checkbox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/Checkbox-mount_base.html">3196</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/Checkbox-mount.html">3212</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>CheckboxBase</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/CheckboxBase-mount_base.html">2799</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/CheckboxBase-mount.html">2835</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ChoiceGroup</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/ChoiceGroup-mount_base.html">5213</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/ChoiceGroup-mount.html">5405</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ComboBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/ComboBox-mount_base.html">1501</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/ComboBox-mount.html">1505</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>CommandBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/CommandBar-mount_base.html">11344</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/CommandBar-mount.html">11386</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ContextualMenu</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/ContextualMenu-mount_base.html">12889</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/ContextualMenu-mount.html">12762</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DefaultButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/DefaultButton-mount_base.html">1640</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/DefaultButton-mount.html">1674</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRow</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/DetailsRow-mount_base.html">4330</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/DetailsRow-mount.html">4347</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowFast</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/DetailsRowFast-mount_base.html">4342</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/DetailsRowFast-mount.html">4415</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowNoStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/DetailsRowNoStyles-mount_base.html">4074</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/DetailsRowNoStyles-mount.html">4237</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Dialog</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/Dialog-mount_base.html">3726</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/Dialog-mount.html">3690</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DocumentCardTitle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/DocumentCardTitle-mount_base.html">687</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/DocumentCardTitle-mount.html">677</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Dropdown</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/Dropdown-mount_base.html">3891</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/Dropdown-mount.html">3830</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusTrapZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/FocusTrapZone-mount_base.html">2411</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/FocusTrapZone-mount.html">2463</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/FocusZone-mount_base.html">2389</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/FocusZone-mount.html">2354</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/GroupedList-mount_base.html">2269</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/GroupedList-mount.html">2586</a></td>
            <td>2</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/GroupedList-mount.regression.txt">Possible regression</a></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/GroupedList-virtual-rerender_base.html">1357</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/GroupedList-virtual-rerender.html">1357</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/GroupedList-virtual-rerender-with-unmount_base.html">1980</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/GroupedList-virtual-rerender-with-unmount.html">2013</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/GroupedListV2-mount_base.html">674</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/GroupedListV2-mount.html">672</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/GroupedListV2-virtual-rerender_base.html">654</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/GroupedListV2-virtual-rerender.html">662</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/GroupedListV2-virtual-rerender-with-unmount_base.html">684</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/GroupedListV2-virtual-rerender-with-unmount.html">670</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>IconButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/IconButton-mount_base.html">2330</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/IconButton-mount.html">2310</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Label</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/Label-mount_base.html">866</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/Label-mount.html">886</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Layer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/Layer-mount_base.html">5173</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/Layer-mount.html">5089</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Link</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/Link-mount_base.html">989</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/Link-mount.html">972</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MenuButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/MenuButton-mount_base.html">2050</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/MenuButton-mount.html">2019</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MessageBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/MessageBar-mount_base.html">2769</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/MessageBar-mount.html">2715</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Nav</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/Nav-mount_base.html">3961</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/Nav-mount.html">3881</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>OverflowSet</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/OverflowSet-mount_base.html">1654</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/OverflowSet-mount.html">1640</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Panel</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/Panel-mount_base.html">3086</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/Panel-mount.html">3073</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/Persona-mount_base.html">1515</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/Persona-mount.html">1521</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Pivot</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/Pivot-mount_base.html">1958</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/Pivot-mount.html">1993</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>PrimaryButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/PrimaryButton-mount_base.html">1828</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/PrimaryButton-mount.html">1840</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Rating</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/Rating-mount_base.html">8226</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/Rating-mount.html">8436</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SearchBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/SearchBox-mount_base.html">1791</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/SearchBox-mount.html">1793</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Shimmer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/Shimmer-mount_base.html">3429</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/Shimmer-mount.html">3397</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Slider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/Slider-mount_base.html">2542</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/Slider-mount.html">2508</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/SpinButton-mount_base.html">5545</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/SpinButton-mount.html">5555</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Spinner</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/Spinner-mount_base.html">969</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/Spinner-mount.html">975</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SplitButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/SplitButton-mount_base.html">3684</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/SplitButton-mount.html">3688</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Stack</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/Stack-mount_base.html">994</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/Stack-mount.html">996</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithIntrinsicChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/StackWithIntrinsicChildren-mount_base.html">2921</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/StackWithIntrinsicChildren-mount.html">2876</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithTextChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/StackWithTextChildren-mount_base.html">5727</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/StackWithTextChildren-mount.html">5832</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchColorPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/SwatchColorPicker-mount_base.html">12640</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/SwatchColorPicker-mount.html">12559</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TagPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/TagPicker-mount_base.html">3146</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/TagPicker-mount.html">3177</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TeachingBubble</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/TeachingBubble-mount_base.html">98310</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/TeachingBubble-mount.html">97013</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Text</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/Text-mount_base.html">950</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/Text-mount.html">943</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TextField</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/TextField-mount_base.html">1912</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/TextField-mount.html">1973</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/ThemeProvider-mount_base.html">1890</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/ThemeProvider-mount.html">1836</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/ThemeProvider-virtual-rerender_base.html">1294</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/ThemeProvider-virtual-rerender.html">1302</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/ThemeProvider-virtual-rerender-with-unmount_base.html">2556</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/ThemeProvider-virtual-rerender-with-unmount.html">2598</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Toggle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/Toggle-mount_base.html">1327</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/Toggle-mount.html">1341</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>buttonNative</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/buttonNative-mount_base.html">671</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25665/perf-test/buttonNative-mount.html">644</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9423"/>

File: change/@fluentui-react-e9e6c992-15eb-4571-803d-e42257153c75.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Always give panel header background color",
```
Comment (khmakoto) [inline change/@fluentui-react-e9e6c992-15eb-4571-803d-e42257153c75.json:3]:
```suggestion
  "comment": "fix: Always give panel header a background color, even before resize.",
```

**Review by khmakoto — APPROVED** [review]

**Comment by fabricteam** [comment]
## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the 5 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=25665&commitId=7b28d5b04de1db5d947103196fa406b8bc78522b&env=prod)

✅ There was 0 screenshots added, 2 screenshots removed, 1032 screenshots unchanged, 0 screenshots with different dimensions and 3 screenshots with visible difference.


  <details><summary><b>unknown</b> 5 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|TagPicker.Root.chromium.png|135| Changed|
|TagPicker.Selected - RTL.chromium.png|135| Changed|
|TagPicker.Selected.chromium.png|135| Changed|
|Pivot - Overflow.Root.Narrow - Last tab selected.chromium.png|0| Removed|
|Pivot - Overflow.Root.Narrow - Overflow menu.chromium.png|0| Removed|

</details>


<div id="vrtCommentfluentuiv8"/>

**Pull request merged**